pkg/api/kubegraph/edges.go

```diff
@@ -125,7 +125,7 @@ func AddAllManagedByControllerPodEdges(g osgraph.MutableUniqueGraph) {
 	}
 }
 
-func AddMountedSecretEdges(g osgraph.Graph, podSpec *kubegraph.PodSpecNode) {
+func AddMountedSecretEdges(g osgraph.Graph, podSpec *kubegraph.PodSpecNode, isResourceKindForbidden bool) {
```

Comment:
Bad name and poorly constructed to be generic.  If you want it generic, pass the entire forbidden map.  If you want it specific, name it specifically.